packages/react/src/auth/ChooseScopeForm.tsx

```diff
@@ -62,10 +62,10 @@ export function ChooseScopeForm(props: ChooseScopeFormProps): JSX.Element {
               ];
             }
             return (
-              <>
+              <div key={scopeName + '_group'}>
```

Comment:
If you don't actually need a `<div>`, you can use `Fragment` (imported `from 'react';`)

That's the syntactic way to add a `key` prop to a fragment.

```suggestion
              <Fragment key={scopeName + '_group'}>
```